src/plugins/intel_gpu/tests/unit/test_cases/convolution_gpu_test.cpp

```diff
@@ -9110,6 +9110,7 @@ class convolution_scale_random_test : public convolution_random_test_base<InputT
 };
 
 class convolution_random_smoke_test : public testing::TestWithParam<convolution_random_test_all_params> {};
+class convolution_random_fsv32_test : public testing::TestWithParam<convolution_random_test_all_params> {};
 
```

Comment:
random spot)
* How does it perform on old platforms with SIMD16? If your new implementation performs good, I'd suggest to remove simd8 implementation. That will simplify the code.
* Does this test code validate simd16 version on old platforms? We don't have precommit test for the new platforms.